openedx_events/management/commands/generate_avro_schemas.py

```diff
@@ -0,0 +1,89 @@
+"""
+Management command to generate and store Avro schemas for testing.
+"""
+import json
+import logging
+import os
+from importlib import import_module
+
+from django.core.management.base import BaseCommand
+
+from openedx_events.event_bus.avro.serializer import AvroSignalSerializer
+from openedx_events.tooling import KNOWN_UNSERIALIZABLE_SIGNALS, OpenEdxPublicSignal, load_all_signals
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command to save current Avro schemas for all signals for schema evolution testing.
+    """
+
+    help = """
+    Generate and save the current Avro schemas of OpenEdxPublicSignals.
+
+    Schemas will be saved in openedx_events/avro/tests/schemas. The folder will be created if it does not already exist.
+    Should only be used for new signals. Existing files will be overwritten, so use with caution.
```

Comment:
Thoughts on making it easier to do the right thing:
```suggestion
    Should only be used for new signals. If a schema is changing, it is recommended you leave the original
    in place. Only overwrite the file for an exceptional case, like the schema still being under development.
```